### ISSUE TITLE
`InvokeParentHookMethodRule` does not detect parent call in `finally`

### ISSUE THREAD

**Issue by devfrey** [ergebnis/phpstan-rules#961]
When a test case calls `parent::tearDown()` from a `finally` clause, the `InvokeParentHookMethodRule` does not detect it. Even though the method will always be called.

```php
protected function tearDown(): void
{
    try {
        // something that might throw an exception
    } finally {
        parent::tearDown();
    }
}
```

```
Method Tests\TestCase::tearDown() does not invoke parent::tearDown().
```

**Comment by MrSuddenJoy** [comment]
Hm....

```php
$rs = new Rule('rule name', 'Rule content goes here');
```

Pretty standard way if you ask me.
